mkdocs.yml

```diff
@@ -114,4 +114,5 @@ nav:
       - Query a Webpage: "examples/query-webpage.md"
     - Misc:
         - Count Tokens: "examples/count-tokens.md"
-        - Using Text Generation Web UI: "examples/using-text-generation-web-ui.md"
\ No newline at end of file
+        - Using Text Generation Web UI: "examples/using-text-generation-web-ui.md"
+        - Using Text Generation Web UI: "examples/dynamodb-conversation.md"
```

Comment:
Can you update the title to something like "Storing Conversation Memory In DynamoDB"?

Reply:
Done!